docs/source/usage/firstread.rst

```diff
@@ -145,7 +145,7 @@ Python
    print("openPMD version: ",
          series.openPMD)
 
-   if( series.contains_attribute("author") )
+   if( series.contains_attribute("author") ):
```

Comment:
Can we even write this? :)
```suggestion
   if series.contains_attribute("author"):
```

Reply:
Yes, of course :) Thanks for checking it!